.github/workflows/tests.yml

```diff
@@ -17,11 +17,11 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: ['ubuntu-latest', 'macos-latest']
-        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12']
+        os: ['ubuntu-22.04', 'macos-latest']
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12', '3.13']
```

Comment:
Can 3.7 be removed? I mean, it is not an active version anymore.

Reply:
I'd say so

Reply:
or maybe we just keep it just in case, since most people are still on 3.8-3.9, so I assume another bulk will still be at 3.7

Reply:
The package is still compatible even with Python 3.6 or even lower versions of Python. Even while we officially don't support it any longer, and despite it's EOL, I don't think we need to drop support on purpose. As many lovely stories go, it might be a lifesaver to some people out there.

Reply:
Good by me

Reply:
As long as it doesn't prevent us from evolving our code base, I have no issues with supported non-active versions.